host/testcluster.go

```diff
@@ -281,8 +283,32 @@ func newArchiverBase(enabled bool, logger log.Logger) *ArchiverBase {
 	}
 }
 
+func (tc *TestCluster) DisableFaultInjection() {
```

Comment:
Instead of DisableFaultInjection, maybe SetFaultInjectionRate(rate float) would be better? So you can enable and disable.